code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -149,3 +149,15 @@
 	volume = 20
 	falloff_distance = 2
 	falloff_exponent = 5
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/datum/looping_sound/pump
+	start_sound = 'sound/machines/pump/pump_start.ogg'
+	start_length = 0.8 SECONDS
+	mid_sounds = 'sound/machines/pump/pump_mid.ogg'
+	mid_length = 1.3 SECONDS
```

Comment:
If at all possible you want sound snippets to be small. Byond will only update "directional" sound when we play a new audio slice, so if they're too long like this you'll get really obvious "jumping"

Reply:
I've shortened it with the latest commit and walking around it sounds much smoother now, thanks for the suggestion